src/Specifications.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/** {@link https://csa-iot.org/developer-resource/specifications-download-request/ Matter Core Specification 1.0} */
```

Comment:
Is that really the URL?

Reply:
Yes, that's the most direct link to download the specs. The specs themselves don't have a direct URL...

Reply:
Ahh, ok so read as This link and then document X ... got it